src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -14,858 +14,1080 @@ namespace System.Runtime.Intrinsics.X86
         /// <summary>
-        /// __m128i _mm_loadl_epi64 (__m128i const* mem_addr)
+        /// __m128i _mm_loadl_epi64 (__m128i const* mem_addr);
+        ///   MOVQ xmm, reg/m64
         /// </summary>
-        public static unsafe Vector128<ulong> LoadScalar(ulong* address) => LoadScalar(address);
-        
+        public static unsafe Vector128<ulong> LoadScalarVector128(ulong* address) => LoadScalarVector128(address);
+
         /// <summary>
-        /// void _mm_maskmoveu_si128 (__m128i a,  __m128i mask, char* mem_address)
+        /// void _mm_maskmoveu_si128 (__m128i a,  __m128i mask, char* mem_address);
+        ///   MASKMOVDQU xmm, xmm
         /// </summary>
         public static unsafe void MaskMove(Vector128<sbyte> source, Vector128<sbyte> mask, sbyte* address) => MaskMove(source, mask, address);
         /// <summary>
-        /// void _mm_maskmoveu_si128 (__m128i a,  __m128i mask, char* mem_address)
+        /// void _mm_maskmoveu_si128 (__m128i a,  __m128i mask, char* mem_address);
+        ///   MASKMOVDQU xmm, xmm
         /// </summary>
         public static unsafe void MaskMove(Vector128<byte> source, Vector128<byte> mask, byte* address) => MaskMove(source, mask, address);
 
         /// <summary>
-        /// __m128i _mm_max_epu8 (__m128i a,  __m128i b)
+        /// __m128i _mm_max_epu8 (__m128i a,  __m128i b);
+        ///   PMAXUB xmm, xmm/m128
         /// </summary>
         public static Vector128<byte> Max(Vector128<byte> left, Vector128<byte> right) => Max(left, right);
         /// <summary>
-        /// __m128i _mm_max_epi16 (__m128i a,  __m128i b)
+        /// __m128i _mm_max_epi16 (__m128i a,  __m128i b);
+        ///   PMAXSW xmm, xmm/m128
         /// </summary>
         public static Vector128<short> Max(Vector128<short> left, Vector128<short> right) => Max(left, right);
         /// <summary>
-        /// __m128d _mm_max_pd (__m128d a,  __m128d b)
+        /// __m128d _mm_max_pd (__m128d a,  __m128d b);
+        ///   MAXPD xmm, xmm/m128
         /// </summary>
         public static Vector128<double> Max(Vector128<double> left, Vector128<double> right) => Max(left, right);
 
         /// <summary>
-        /// __m128d _mm_max_sd (__m128d a,  __m128d b)
+        /// __m128d _mm_max_sd (__m128d a,  __m128d b);
+        ///   MAXSD xmm, xmm/m64
         /// </summary>
         public static Vector128<double> MaxScalar(Vector128<double> left, Vector128<double> right) => MaxScalar(left, right);
 
         /// <summary>
-        /// __m128i _mm_min_epu8 (__m128i a,  __m128i b)
+        /// void _mm_mfence(void);
+        ///   MFENCE
+        /// </summary>
+        public static void MemoryFence() => MemoryFence();
+
+        /// <summary>
+        /// __m128i _mm_min_epu8 (__m128i a,  __m128i b);
+        ///   PMINUB xmm, xmm/m128
         /// </summary>
         public static Vector128<byte> Min(Vector128<byte> left, Vector128<byte> right) => Min(left, right);
         /// <summary>
```

Comment:
nit: missing empty line

Reply:
Missing newline where?

Reply:
before line 835 - usually we have one empty line between declarations but sometime they are "missing"

Reply:
There are a lot of intrinsics which currently only have newlines between logical groupings (e.g. after all Min methods, there is a newline, or after all Add methods, there is a newline).

Reply:
OK - it is not visible that well in diffs

Reply:
agreed. But it is the current "convention".

I'd rather fix it up in a separate PR, if we want to do so.

Reply:
Yeah, it is small editorial detail and we will have some opportunities to fix it soon